test/commands/addons/info.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+/* globals describe context it beforeEach */
+
+let fixtures = require('../../fixtures')
+let util = require('../../util')
+let cli = require('heroku-cli-util')
+let nock = require('nock')
+let cmd = require('../../../commands/addons/info')
+
+describe('addons:info', function () {
+  beforeEach(function () {
+    cli.mockConsole()
+    nock.cleanAll()
+  })
+
+  context('with add-ons', function () {
+    beforeEach(function () {
+      nock('https://api.heroku.com', {reqheaders: {'Accept-Expansion': 'addon_service,plan'}})
+        .get('/addons/www-db')
+        .reply(200, fixtures.addons['www-db'])
+      nock('https://api.heroku.com')
+        .get(`/addons/${fixtures.addons['www-db']['id']}/addon-attachments`)
+        .reply(200, [fixtures.attachments['acme-inc-www::DATABASE']])
+    })
+
+    it('prints add-ons in a table', function () {
+      return cmd.run({flags: {}, args: {addon: 'www-db'}}).then(function () {
+        util.expectOutput(cli.stdout,
+`=== www-db
+Attachments:  acme-inc-www::DATABASE
+Installed at: Invalid Date
+Owning app:   acme-inc-www
+Plan:         heroku-postgresql:hobby-dev
+Price:        free
+`)
+      })
+    })
+  })
+
+  context('with app add-ons', function () {
+    beforeEach(function () {
+      nock('https://api.heroku.com', {reqheaders: {'Accept-Expansion': 'addon_service,plan'}})
+        .get('/apps/example/addons/www-db')
+        .reply(200, fixtures.addons['www-db'])
+      nock('https://api.heroku.com')
+        .get(`/addons/${fixtures.addons['www-db']['id']}/addon-attachments`)
+        .reply(200, [fixtures.attachments['acme-inc-www::DATABASE']])
+    })
+
+    it('prints add-ons in a table', function () {
```

Comment:
Ah! Cool, yup, I did miss the difference in the URLs that were being mocked! Thank you for the explanation here.

Yeah, the additional test case is good to confirm.